pkgs/tools/misc/otel-cli/default.nix

```diff
@@ -1,30 +1,42 @@
-{ lib, bash, buildGoModule, fetchFromGitHub, getent, stdenv }:
-
+{
```

Comment:
While this is not a blocker for me, I think that reformatting could be controversial.

Reply:
yeah, follow the standard elsewhere; no issue with these being broken up to multiline.